src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/editprofile/AvatarComponentObject.java

```diff
@@ -2,23 +2,20 @@
 
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)